db/models/index.js

```diff
@@ -18,28 +18,23 @@ const Photo = require('./photo');
 const BrewType = require('./brewType');
 const Unit = require('./unit');
 
-  // I'm assuming that the following will be added thru associations (see index).
-  // BrewID... unique brew types... each product can have one
-  // SellerID... one seller per product
-  // Photos... multiple to one product entry
-
 OAuth.belongsTo(User);
 User.hasOne(OAuth);
 
-Product.belongsTo(Seller); //sellerID on product
-Seller.hasMany(Product); //symmetrical
+Product.belongsTo(Seller, {as: 'seller'}); //sellerID on product
+Seller.hasMany(Product, {as: 'product'}); //symmetrical
 
-Seller.hasMany(Photo); //sellerID on photo
+Seller.hasMany(Photo, {as: 'seller'}); //sellerID on photo
 
 //symetrical associations?
-Product.belongsTo(BrewType); //brewID on product
-Product.belongsTo(Unit); //unitID on product
+Product.belongsTo(BrewType, {as: 'brew'}); //brewID on product
+Product.belongsTo(Unit, {as: 'unit'}); //unitID on product
 
-Product.hasMany(Photo); //productID on photo
+Product.hasMany(Photo, {as: 'product'}); //productID on photo
 
 //reviews - created from/after product by logged-in user
-Review.belongsTo(Product); //productID on review
-Review.belongsTo(User); //userID on review
+Review.belongsTo(Product, {as: 'product'}); //productID on review
+Review.belongsTo(User, {as: 'user'}); //userID on review
```

Comment:
Question: should we enable photo uploads for reviews (and create an association between photo and review)?